mocha.css

```diff
@@ -212,10 +220,12 @@ body {
   text-align: center;
   background: #eee;
   font-size: 15px;
-  -moz-border-radius: 15px;
-  border-radius: 15px;
-  -webkit-transition: opacity 200ms;
-  -moz-transition: opacity 200ms;
+  -webkit-border-radius: 15px;
+     -moz-border-radius: 15px;
+          border-radius: 15px;
+  -webkit-transition:opacity 200ms;
+  -moz-transition:opacity 200ms;
+  -o-transition:opacity 200ms;
```

Comment:
That's a good question; I was respecting the opera reference in another line (` line 99   -o-border-radius: 5px;`); It's not consistent with other references to border-radius, but I erred on the side of caution.  Lowest common denominator, if the file supports vendor prefixes for the era of Opera 11.5 --when Opera standardized their css for  border radius and lost the need for the -o-prefix; at that time, the -o- prefix for transition was also required (until version 15) for o-transition, but not for box-shadow. Opera is now at version 38, but as mocha is so widely used, I didn't want to _remove_ support, but _ensure_ it.

I had go to cast a wide net on [the settings at autoprefixer](https://autoprefixer.github.io/) for the Opera settings to show up, using `> .04%, last 30 versions` (Browsers with more than .04% share, plus back 30 versions). It's likely safe to remove the prefix considering the developer audience.  Supporting the last 10 versions with browsers more than 1% share, it's now safe to drop most -moz and -o- prefixes.